src/glue/jobs/compare_parquet_datasets.py

```diff
@@ -1,25 +1,25 @@
-import argparse
 import datetime
```

Comment:
I recommend using isort to handle import sort order: https://pycqa.github.io/isort/docs/configuration/pre-commit.html

It could be part of the pre-commit to do this for you.

Reply:
Thanks! Handy, just added it